assets/presto_clp/main.py

```
@@ -0,0 +1,109 @@
+        return self.docker_execute(f'/home/presto/presto-cli/target/presto-cli-0.293-executable.jar --catalog clp --schema default --execute "{query}"', check)
+
+    def sql_execute(self, query, check=True):
+        if query[-1] != ';':
+            query = query+';'
+        return self.docker_execute(f"mysql -h {HOST_IP} -P 6001 -u clp-user -p{SQL_PASSWORD} -e '{query}' clp-db", check)
+
+    def ingest(self):
+        """
+        Ingests the dataset at self.datasets_path
+        """
+        os.system(f'{CLP_PRESTO_HOST_STORAGE}/sbin/compress.sh --timestamp-key {self.timestamp} {self.datasets_path_in_host}')
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)